script/build-rest.ejs

```diff
@@ -96,7 +96,7 @@ function createClient(auth, opts) {
  * var flickr = new Flickr(process.env.FLICKR_API_KEY);
  *
  * flickr.photos.search({
- *   photo_id: 25825763 // sorry, @dokas
+ *   text: 'doggo'
```

Comment:
https://www.flickr.com/photos/140711719@N03/27804758406/in/photolist-Jn1DYj-RUyyG6-U4QGTP-S7RevM-THrENd-UuMmtC-Wc27mZ-UhfuFd-UUKRek-VD4W3h-UkTHKM-TvQMux-TywDsm-Uo5h4G-T7f27v-TU4HTV-WhU67g-SBCfJ3-TNsL3h-P4Zrx9-WCKKrS-V8Hmri-TKVDL9-VXTfMH-X8aXms-VW21Ae-VWvTmT-UHJpbZ-opEggf-V8NjD7-W7M4r2-S3sczq-VGWL7F-Qb84LR-RQEon5-RJAeDL-TzjtW2-V6QbtR-RTWmMm-VjtB7v-Vnpn1W-UUVuhh-SSRpjn-T4bghk-W3C3oh-V7difa-VeLVJX-WCXA8B-Uhfwnu-STJV8o